master/internal/stream/models.go

```diff
@@ -39,12 +39,12 @@ type ModelMsg struct {
 	Description     string    `bun:"description" json:"description"`
 	Archived        bool      `bun:"archived" json:"archived"`
 	CreationTime    time.Time `bun:"creation_time" json:"creation_time"`
-	Notes           JSONB     `bun:"notes" json:"notes"`
+	Notes           string    `bun:"notes" json:"notes"`
```

Comment:
question (non-blocking): Is there a reason we're moving way from JSONB type?

Reply:
`Notes` is text type in DB, so I think `string` is more accurate

Reply:
Makes sense!